src/FormatCommand.cs

```diff
@@ -75,6 +77,11 @@ internal static RootCommand CreateCommandLineOptions()
                 {
                     IsHidden = true
                 },
+                new Option(new[] {"--binarylog", "-bl" }, Resources.Log_all_project_or_solution_load_information_to_a_binary_log_file)
```

Comment:
Although `-bl` is what msbuild uses, I think we want to stick to single letter short names.

Reply:
I wouldn't bother giving this one a short name
```suggestion
                new Option(new[] { "--binarylog" }, Resources.Log_all_project_or_solution_load_information_to_a_binary_log_file)
```